### PR TITLE
Add another section with untriaged issues

### PR DESCRIPTION
**Pull request by jan-auer**
These are issues without labels, assignees (and if feasible: comments from one of the org members).

### CODE REVIEW

**Comment by jan-auer** [comment]
When deploying this, we need to wipe the config again...

File: lib/defaults.js [hunk 1]
```diff
@@ -10,28 +11,35 @@ module.exports = {
   email: {
     sender: '"🤖 Eos - Github Bot" <noreply@md.getsentry.com>',
     subjectTemplate: 'Attention: <%= count %> Pull requests',
```
Comment (HazAT) [inline lib/defaults.js]: The subject doesn't fit that well anymore.
How about `Hey! Github needs your attention`

File: lib/defaults.js [hunk 2]
```diff
@@ -10,28 +11,35 @@ module.exports = {
...
+    toReviewTemplate: `<p>These pull requests need to be <b>reviewed</b>:</p>
       <ul>
         <%= issues %>
       </ul>`,
-    toCompleteTemplate: `<p>These pull requests need to be handled:</p>
+    toCompleteTemplate: `<p>These pull requests need to be <b>handled</b>:</p>
+      <ul>
+        <%= issues %>
+      </ul>`,
+    newIssuesTemplate: `<p>There are also issues you could <b>label and assign</b>:</p>
```
Comment (HazAT) [inline lib/defaults.js]: I would not write `also`, since it could be that there are new issues.

File: lib/mailer.js [hunk 3]
```diff
@@ -129,30 +153,16 @@ module.exports = class Mailer {
...
-        pullRequests: prsToReview,
-      }),
-    }) : '';
-
-    const toComplete = prsToComplete.length > 0 ? this.templates.toCompleteTemplate({
-      issues: this.templates.issueTemplate({
-        pullRequests: prsToComplete,
-      }),
-    }) : '';
+    const subject = this.templates.subjectTemplate({ count: report.count() });
```
Comment (HazAT) [inline lib/mailer.js]: If we change the subject, we need to change this here.

**Pull request merged**